build.gradle

```diff
@@ -100,52 +109,34 @@ task prepareControlFile(type: Copy) {
 }
 
 debian {
-  packagename = "camus-compressor"
-  controlDirectory = "build/package"
-  changelogFile = "src/package/changelog"
+  packagename = 'camus-compressor'
+  controlDirectory = 'build/package'
+  changelogFile = 'src/package/changelog'
 
   data {
-    def baseDir = "build"
+    def baseDir = 'build'
     file {
       name = "${baseDir}/libs/camus-compressor-${project.version}-all.jar"
       target = "usr/lib/hadoop-tools/camus-compressor/compressor-${project.version}.jar"
       mapper {
-        fileMode = "644"
+        fileMode = '644'
       }
     }
     file {
-      name = "src/main/resources/compressor.sh"
-      target = "usr/lib/hadoop-tools/camus-compressor/compressor.sh"
+      name = 'src/main/resources/compressor.sh'
+      target = 'usr/lib/hadoop-tools/camus-compressor/compressor.sh'
       mapper {
-        fileMode = "755"
+        fileMode = '755'
       }
     }
     link {
-      path = "usr/lib/hadoop-tools/camus-compressor/compressor.jar"
+      path = 'usr/lib/hadoop-tools/camus-compressor/compressor.jar'
       name = "/usr/lib/hadoop-tools/camus-compressor/compressor-${project.version}.jar"
     }
   }
 }
 
 
-/***** integration tests *****/
-
-sourceSets {
-    integration {
-        compileClasspath += sourceSets.test.compileClasspath
-        runtimeClasspath += sourceSets.test.runtimeClasspath
-        java.srcDir file('src/integration/java')
-    }
-}
-
-task integration(type: Test) {
-    testClassesDir = sourceSets.integration.output.classesDir
-    classpath = project.sourceSets.main.output +
-       project.sourceSets.integration.output +
-       sourceSets.test.compileClasspath
-    
-    dependsOn integrationClasses
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.3'
```

Comment:
since you change it, maybe lets move to some less ancient version like `2.13` ;)